contentcuration/contentcuration/utils/publish.py

```diff
@@ -151,7 +151,7 @@ def create_content_database(channel, force, user_id, force_exercises, progress_t
         if progress_tracker:
             progress_tracker.track(90)
         map_prerequisites(channel.main_tree)
-        save_export_database(channel.pk)
+        save_export_database(channel.pk, channel.version)
```

Comment:
This works perfectly except for one thing - the channel version gets incremented _after_ this function is called (look above to see where `create_content_database` and `increment_channel_version` are called respectively!)

So, here you need to instead pass in `channel.version + 1` and add a code comment as to why we are incrementing it here.

Reply:
You will see similar logic happening inside the `map_channel_to_kolibri_channel` function where the version written into the database is incremented by one also.

Reply:
I've made the adjustments to pass in `channel.version + 1` as suggested and added a comment in the code to explain why we're incrementing the version at this point. Thank you for pointing that out!

Reply:
Thanks, I'll test locally, and then hopefully this should be good to merge!